2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,51 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  let allCityTemperature = [];
+  for (const city of cities) {
+    let cityTemperature = temperatureService(city);
+    allCityTemperature.push(
+      `The temperature in ${city} is ${cityTemperature} degrees`
+    );
+  }
+  return allCityTemperature;
```

Comment:
@lalehkoupa  well done :)

Reply:
@Thanks a lot @ellietms for reviewing my codes:)